package.json

```diff
@@ -8,7 +8,7 @@
     "build": "next build",
     "start": "npx prisma generate && next start",
     "lint": "next lint",
-    "test": "jest --watch",
+    "test": "docker-compose up -d db && npx prisma migrate dev && jest && docker-compose down",
```

Comment:
Does it destroy the test db though?

Reply:
Oh but I mean in terms of data, does the test db persist? It seems not worth to have past data from tests interfere with new tests. I think every time we run tests the test db should be empty